enterprise/backend/test/metabase_enterprise/sandbox/api/gtap_test.clj

```diff
@@ -54,6 +54,33 @@
                                         :group_id             group-id
                                         :card_id              card-id
                                         :attribute_remappings {"foo" 1}}))))))))
+
+(deftest fetch-gtap-test
+  (testing "GET /api/mt/gtap/"
+    (with-gtap-cleanup
+      (mt/with-temp* [Table                  [{table-id-1 :id}]
+                      Table                  [{table-id-2 :id}]
+                      PermissionsGroup       [{group-id-1 :id}]
+                      PermissionsGroup       [{group-id-2 :id}]
+                      Card                   [{card-id :id}]
+                      GroupTableAccessPolicy [{gtap-id-1 :id} {:table_id table-id-1
+                                                               :group_id group-id-1
+                                                               :card_id  card-id}]
+                      GroupTableAccessPolicy [{gtap-id-2 :id} {:table_id table-id-2
+                                                               :group_id group-id-2
+                                                               :card_id  card-id}]]
+        (testing "Test that we can fetch the list of all GTAPs"
+          (is (partial=
+               [{:id gtap-id-1 :table_id table-id-1 :group_id group-id-1}
+                {:id gtap-id-2 :table_id table-id-2 :group_id group-id-2}]
+               (mt/user-http-request :crowberto :get 200 "mt/gtap/"))))
```

Comment:
It seems like this test would fail if you already have other GTAPs in your database, right? In interests of making this REPL-friendly I would filter out the results so you're only matching ones where ID matches `gtap-id-1` or `gtap-id-2`. If the endpoint returns those two objects it should be fine, and we can ignore the other ones it returns